assets/state-mig-manager/0420_configmap.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
     app: nvidia-mig-manager
 data:
   entrypoint.sh: |-
-    #!/bin/bash
+    #!/bin/sh
```

Comment:
We're not yet switching to distroless for the mig-manager, or has work started to also port its bash script to shell?

Reply:
We haven't switched mig-manager to distroless yet, but we were planning to start that effort soon -- either by porting the bash script to shell or rewriting it in Go.

Regardless, updating the entrypoint script to use `sh` now (before the switch to distroless) shouldn't be an issue, right?